src/module-elasticsuite-core/view/adminhtml/web/template/system/upsell-notification.html

```diff
@@ -0,0 +1,12 @@
+<div class="message elasticsuite-upsell-content" data-bind="visible: visible">
+    <!-- Dismiss button -->
+    <button type="button"
+            class="action-close"
+            data-bind="click: validateNotification"
+            style="position: absolute; top: 10px; right: 10px; background: none; border: none; font-size: 18px; cursor: pointer;"
+            aria-label="Close">
+    </button>
+    <p><em>You’re currently using the ElasticSuite Open Source Edition.</em></p>
+    <p>Unlock AI-powered, advanced features like <strong>bestseller boosting, behavioral merchandising, automated recommendations, A/B testing...</strong> with the ElasticSuite Premium Edition.</p>
+    <p><strong><a href="https://elasticsuite.io/elastic-suite-features/">➜ Explore the full feature list</a></strong></p>
```

Comment:
Add something to the tune of "and do not hesitate to contact us to know more" or "reach out to us" (with the intended meaning being "through the site") ?

Reply:
(Will an actual local test later)

Reply:
Done (and also added i18n which was missing)